shared/Cargo.lock

```diff
@@ -0,0 +1,476 @@
+# This file is automatically @generated by Cargo.
```

Comment:
@Tguntenaar suggest adding this crate to members of wokspace https://github.com/NEAR-DevHub/neardevhub-contract/blob/main/Cargo.toml#L2, so that it doesn't introduce a `Cargo.lock` of its own